docs/source/overview/api.rst

```diff
@@ -175,6 +175,55 @@ Note that a tag is automatically created for the resource with the name ``<names
                 * ``resource_name`` : name of the resource to be created in the Paraglider controller (note: this name will be scoped on cloud and namespace when stored)
                 * ``description``: JSON string describing the resource to be created (excluding networking details)
 
+Attach
```

Comment:
Could we also add a warning callout that states this is for Azure only at the moment and other clouds are WIP?

As a tangent, do the other clouds return gracefully at the moment when using this?

Reply:
Added the note that it's only supported by Azure currently. The attach API uses the create resource API route, so it would return gracefully. However, the error message may not always be "not supported". I'll create an issue for this.